lib/node_modules/@stdlib/blas/ext/base/gcusumkbn2/lib/ndarray.js

```diff
@@ -67,6 +71,11 @@ function gcusumkbn2( N, sum, x, strideX, offsetX, y, strideY, offsetY ) {
 	if ( N <= 0 ) {
 		return y;
 	}
+	ox = arraylike2object( x );
+	oy = arraylike2object( y );
+	if ( ox.accessorProtocol || oy.accessorProtocol ) {
+		return accessors( N, sum, ox, strideX, offsetX, oy, strideY, offsetY );
```

Comment:
```suggestion
	accessors( N, sum, ox, strideX, offsetX, oy, strideY, offsetY );
	return y;
```

You need to return the original output array, not the wrapped object.